packages/core/src/components/forms/_control-group.scss

```diff
@@ -106,7 +106,6 @@ Styleguide control-group
 
     &:focus {
       // establish new stacking context so focus state covers neighbors
-      position: relative;
```

Comment:
hm, what do you mean there was already a new stacking context? I don't see one for these selectors (L100-102).

we probably want to try removing this line (and the commend above) and adding `@include new-render-layer();` right after the `.#{$ns}-select select {` line (L102)

Reply:
Got it!